packages/client/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@slack/client",
+  "version": "5.0.0",
+  "description": "Legacy wrapper for official Slack Platform's Web API, RTM API, and Incoming Webhook libraries. Use @slack/web-api, @slack/rtm-api, or @slack/webhook instead.",
+  "author": "Slack Technologies, Inc.",
+  "license": "MIT",
+  "keywords": [
+    "slack",
+    "web-api",
+    "rtm-api",
+    "bot",
+    "client",
+    "http",
+    "websocket",
+    "request",
+    "api",
+    "proxy",
+    "rate-limiting",
+    "pagination",
+    "state",
+    "connection"
+  ],
+  "main": "dist/index.js",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist/**/*"
+  ],
+  "engines": {
+    "node": ">= 8.9.0",
+    "npm": ">= 5.5.1"
+  },
+  "repository": "slackapi/node-slack-sdk",
+  "homepage": "https://slack.dev/node-slack-sdk/tutorials/migrating_to_v5/",
```

Comment:
☝️ This URL leads to a 404

Reply:
Correct URL: https://slack.dev/node-slack-sdk/tutorials/migrating-to-v5/

Reply:
Fixed in https://github.com/slackapi/node-slack-sdk/pull/761

We’ll likely be releasing that today.